make/dev.mk

```diff
@@ -1,37 +1,37 @@
+ifndef DEV_MK
+DEV_MK:=# Prevent repeated "-include".
+
+include ./make/verbose.mk
+include ./make/git.mk
+
 DOCKER_REPO?=quay.io/openshiftio
 IMAGE_NAME?=devconsole-operator
-SHORT_COMMIT=$(shell git rev-parse --short HEAD)
-ifneq ($(GITUNTRACKEDCHANGES),)
-SHORT_COMMIT := $(SHORT_COMMIT)-dirty
-endif
 
 DEVOPSCONSOLE_OPERATOR_IMAGE?=quay.io/redhat-developers/devopsconsole-operator
 TIMESTAMP:=$(shell date +%s)
-TAG?=$(SHORT_COMMIT)-$(TIMESTAMP)
-
-DEPLOY_DIR:=deploy
+TAG?=$(GIT_COMMIT_ID_SHORT)-$(TIMESTAMP)
 
 .PHONY: create-resources
 create-resources:
-	@echo "Logging using system:admin..."
-	@oc login -u system:admin
-	@echo "Creating sub resources..."
-	@echo "Creating CRDs..."
-	@oc create -f $(DEPLOY_DIR)/crds/devopsconsole_v1alpha1_gitsource_crd.yaml
-	@echo "Creating Namespace"
-	@oc create -f $(DEPLOY_DIR)/namespace.yaml
-	@echo "oc project codeready-devconsole"
-	@oc project codeready-devconsole
-	@echo "Creating Service Account"
-	@oc create -f $(DEPLOY_DIR)/service_account.yaml
-	@echo "Creating Role"
-	@oc create -f $(DEPLOY_DIR)/role.yaml
-	@echo "Creating RoleBinding"
-	@oc create -f $(DEPLOY_DIR)/role_binding.yaml
+	$(info Logging using system:admin...)
+	$(Q)oc login -u system:admin
+	$(info Creating sub resources...)
+	$(info Creating CRDs...)
+	$(Q)oc create -f ./deploy/crds/devopsconsole_v1alpha1_gitsource_crd.yaml
+	$(info Creating Namespace)
+	$(Q)oc create -f ./deploy/namespace.yaml
```

Comment:
I think we were trying to avoid using relative paths - please see the following comment: https://github.com/redhat-developer/devconsole-operator/pull/34#discussion_r268008097

Reply:
@tinakurian There is no reason to not use relative paths as I‘ve outlined in the description of the PR. We are not building out of source, so all is good.